sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -312,10 +312,12 @@ SIREPO.app.directive('cancelledDueToTimeoutAlert', function(authState) {
         template: [
             '<div data-ng-if="simState.getCancelledAfterSecs()" class="alert alert-warning" role="alert">',
               '<h4 class="alert-heading">Cancelled: Maximum runtime exceeded</h4>',
-              '<p>Your simulation ran for {{getTime()}}. To increase your maximum runtime please upgrade to <a href="https://radiasoft.net/sirepo" target="_blank">Sirepo {{ premiumOrEnterprise() }}</a>.</p>',
+              '<p>Your simulation ran for {{getTime()}}. To increase your maximum runtime please upgrade to <a href="https://radiasoft.net/sirepo" target="_blank">Sirepo {{ authState.premiumOrEnterprise() }}</a>.</p>',
```

Comment:
The plan is "Sirepo Premium". This string should be in the schema, and looked up based on the servicePlan or something like that in authState. 

`https://radiasoft.net/sirepo` should not be hardwired here. It should be a looked up in the schema. #2652 needs to be addressed, and it's basically the same idea. We need to name this redirect so that we know 1) that the redirect happens here, and 2) we can control it for on premise installs.

Reply:
Is this right? https://github.com/radiasoft/sirepo/pull/2690/commits/68d5a2d040d51f8ae5ba41517c9b0571c45a1f72

Reply:
https://github.com/radiasoft/sirepo/blob/6d285a3f4e33e9f4871aa63417f5451f8947d4a0/sirepo/job_supervisor.py#L234

Reply:
Seems fine to me. The chaos of it bothers me, but at this stage, I don't see a way out of it.

Reply:
I should be pointing at sirepo.com and not at radiasoft.net

Reply:
Payments go through radiasoft.net/sirepo

Reply:
@e-carlin Please change upgradePlanLink  to /en/plans.html

Reply:
sirepo.com/en/plans.html ? I get a 404 . On radiasoft.net I get redirected to the home page.